src/Input/index.js

```diff
@@ -80,7 +81,13 @@ const Input = ({
         {(endIconClass || showClearButton) && endIconJsx}
         {endContent && <div>{endContent}</div>}
       </div>
-      <Error error={error} />
+      <div className="nds-input-subline">
+        {/* this is in opposite direction to 1. make it easier to render and 2. accommodate screen reading order better */}
```

Comment:
```suggestion
        {/* this is styled using row-reverse to 1. make it easier to render and 2. accommodate screen reading order better */}
```
Feel free to ignore! I wasn't sure what "in opposite direction" meant until I looked at the CSS